src/main/resources/site/parts/statistics/statistics.es6

```diff
@@ -43,6 +49,13 @@ const renderPart = (req) => {
   const phrases = getPhrases(page)
   moment.locale(page.language ? page.language : 'nb')
   const statistic = page.data.statistic && getStatisticByIdFromRepo(page.data.statistic)
+  const wait = app.config && app.config['ssb.statistics.publishWait'] ? parseInt(app.config['ssb.statistics.publishWait']) : 100
+  const maxWait = app.config && app.config['ssb.statistics.publishMaxWait'] ? parseInt(app.config['ssb.statistics.publishMaxWait']) : 10000
+  let waitedFor = 0
+  while (currentlyWaitingForPublish(page) && waitedFor < maxWait) {
+    waitedFor += wait
+    sleep(wait)
```

Comment:
Tror du det er en god ide å gjøre f.eks en "log.warn("Waited for N seconds while publishing") slik at vi kan dobbeltsjekke i loggene at det ikke forekommer oftere enn vi forventer, eller evnt se hvor lenge folk må vente?